src/libs/ajax/Metrics.ts

```diff
@@ -40,6 +40,9 @@ export const Metrics = (signal?: AbortSignal) => {
       isRegistered || signInStatus === 'authenticated'
         ? {
             authProvider: getTerraUser().idp,
+            userInstitute: getTerraUserProfile().institute,
+            userTitle: getTerraUserProfile().title,
+            userDepartment: getTerraUserProfile().department,
```

Comment:
We jump through a lot of hoops (Bard) to avoid sending identifying information to MixPanel.

Reply:
Yeah I was also concerned about that, but in the ticket to add those fields I specifically requested approval from compliance and Sarah Tahiri okayed it so this should be ok (see comments on this ticket)  https://broadworkbench.atlassian.net/browse/ID-1241